forecasters/armadillo/DESCRIPTION

```diff
@@ -0,0 +1,27 @@
+Package: armadillo
+Title: Armadillo forecaster (Larry and Valerie's mobility model)
+Version: 0.0.0.9000
+Authors@R: 
+    person(given = "First",
+           family = "Last",
+           role = c("aut", "cre"),
+           email = "first.last@example.com",
+           comment = c(ORCID = "YOUR-ORCID-ID"))
+Description: Armadillo forecaster implements Larry and Valerie's mobility model. It works with the package evalcast. Now this forecaster only returns point forecasts, no quantile forecasts. 
+License: What license it uses
```

Comment:
Need a license. Easiest is to open the package in Rstudio and run `usethis::use_mit_license()`